### ISSUE TITLE
ng update @nrwl/cypress 8.1 to 8.2 output error "Invalid rule result: Instance of class Promise"

### ISSUE THREAD

**Issue by yglin** [nrwl/nx#1597]
TL;DR: Is there any way, or document, to show me how to at least manually migrate the legacy configuration for cypress ?

run this command 
`ng update @nrwl/cypress --from 8.1.0 --to 8.2.0 --migrate-only`

gives me this output
```
Using package manager: 'npm'
Collecting installed dependencies...
Found 74 dependencies.
    ** Executing migrations for package '@nrwl/cypress' **
Invalid rule result: Instance of class Promise.
```

Please answer the following questions for yourself before submitting an issue.
**YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ x ] I am running the latest version
- [ x ] I checked the documentation and found no answer
- [ x ] I checked to make sure that this issue has not already been filed
- [ x ] I'm reporting the issue to the correct repository (not related to Angular, AngularCLI or any dependency)

## Expected Behavior

the migration complete successfully.

## Current Behavior

Output error **Invalid rule result: Instance of class Promise.** then stop

## Failure Information (for bugs)

There is not much detail except the error message above

### Steps to Reproduce

Just done upgrading nx from 7 to 8, over an existing project.

I did it roughly follow this order:
1. **npm run update** - to update nx
2. **ng update --all --force** - to update Angular ...actually the error happened in this step but I ignored it then.
3. delete the **node_modules** folder and do a clean **npm install**.
4. **npm audit fix** and some dependencies houseclean blah... blah... 
5. **npm start** to check if it runs, well, it runs, and seems normal, nothing crazy.
6. **npm run e2e -- [my-project-e2e]** gives this warning message
```
Warning:
You are using the legacy configuration for cypress.
Please run "ng update @nrwl/cypress --from 8.1.0 --to 8.2.0 --migrate-only".
```
7. Run the **ng update...** command then error

### Context

- version of Nx used
No "@nrwl/nx" package
All "@nrwl/[feature]" packages 8.2.0

- version of Angular CLI used
@angular/cli@8.1.2 

- `angular.json` configuration
- version of Angular DevKit used
    "@angular-devkit/build-angular": "^0.801.2",
    "@angular-devkit/build-ng-packagr": "~0.801.2",

- 3rd-party libraries and their versions
cypress@3.4.0

## Other

Have googled the error for a while, found something seems to related to **schematics** or **angular-cli**, can't be sure and those solutions didn't work.
Actually, the e2e test still run regardless of the warning so this could be a minor issue.
Thanks for reading through.

**Comment by yglin** [comment]
Solved it myself by update **@angular-devkit/core** and **@angular-devkit/schematics** both to 8.1.2
